manticore/core/state.py

```diff
@@ -415,6 +426,8 @@ def concretize(self, symbolic, policy, maxcount=7):
             else:
                 # We assume the path constraint was feasible to begin with
                 vals = (True,)
+        elif policy == "EXPLICIT":
+            vals = explicit_values[:maxcount]
```

Comment:
Note. This _could_ be the source of not feasible states. We are trusting API user to provide valid solutions. This is different to the other policies.

Reply:
That's true. 
I can add solver checks to ensure feasibility for each explicit value. It should't cause too much overhead compared to all the computation required by forking.

Reply:
I'm not opposed to adding optional feasibility checking functionality, but I don't think it's necessary for this PR. The policy API is buried fairly deep, and users are unlikely to stumble over this new policy. 

What I _would_ like to see is `_ValidPolicies` replaced with an Enum so that we can document each of the variants individually. That also doesn't have to be in this PR (since making it backwards compatible will require adding a bunch of type checking) but we should do it soon because I don't think any of the policies are documented right now.